src/components/fundamentals/icons/cart-icon/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
```

Comment:
todo: HTML attributes should be camelCased for consistency

Reply:
Good catch, didn't notice the discrepancy between the export and the other icon files I was comparing to! Done.